tests/integration/online_store/test_universal_online.py

```diff
@@ -0,0 +1,115 @@
+import random
+import unittest
+
+import pandas as pd
+
+from tests.integration.feature_repos.test_repo_configuration import (
+    Environment,
+    parametrize_online_test,
+)
+
+
+@parametrize_online_test
```

Comment:
A pain point for me is that I can't see how this test is being parameterized. What are all the test cases we are going to use to test e2e consistency? It seems hidden from me as a test author. Ideally somebody would just need to look at this single file to see how a test would be parameterized. It's highly unlikely that all the different test files would use the exact same set of parameters (which could benefit from generalization/reuse), so it seems more natural to put the cases here.